config/default/translations/messages-en.properties

```diff
@@ -131,4 +131,428 @@ task.pnc.danger_sign_followup_mother.title = PNC danger sign follow up - mother
+report.pregnancy_danger_sign_follow_up.danger_signs.breathlessness = Breathlessness
+report.pregnancy_danger_sign_follow_up.danger_signs.r_danger_sign_present = Danger signs
+report.pregnancy_danger_sign_follow_up.danger_signs.custom_translations = Custom Translations
+report.pregnancy_danger_sign_follow_up.danger_signs.custom_translations.custom_woman_label_translator = Woman label translator
+report.pregnancy_danger_sign_follow_up.danger_signs.custom_translations.custom_woman_label = Woman label
+report.pregnancy_danger_sign_follow_up.danger_signs.custom_translations.custom_woman_start_label_translator = Woman Start Label translator
+report.pregnancy_danger_sign_follow_up.danger_signs.custom_translations.custom_woman_start_label = Woman start label
+report.pregnancy_danger_sign_follow_up.pregnancy_uuid_ctx = Pregnancy UUID
+report.pregnancy_danger_sign.pregnancy_uuid_ctx = Pregnancy UUID
+report.pregnancy_danger_sign.danger_signs = Danger Sign Check
+report.pregnancy_danger_sign.danger_signs = Danger Sign Check
+report.pregnancy_danger_sign.danger_signs.danger_signs_note = Patient to monitor these danger signs throughout the pregnancy.
+report.pregnancy_danger_sign.danger_signs.danger_signs_question_note = Has any of these danger signs
+report.pregnancy_danger_sign.danger_signs.vaginal_bleeding = Vaginal Bleeding
+report.pregnancy_danger_sign.danger_signs.fits = Fits
+report.pregnancy_danger_sign.danger_signs.severe_abdominal_pain = Severe abdominal pain
+report.pregnancy_danger_sign.danger_signs.severe_headache = Severe headache
+report.pregnancy_danger_sign.danger_signs.very_pale = Very pale
+report.pregnancy_danger_sign.danger_signs.fever = Fever
+report.pregnancy_danger_sign.danger_signs.reduced_or_no_fetal_movements = Reduced or no fetal movements
+report.pregnancy_danger_sign.danger_signs.breaking_water = Breaking of water
+report.pregnancy_danger_sign.danger_signs.easily_tired = Getting tired easily
+report.pregnancy_danger_sign.danger_signs.face_hand_swelling = Swelling of face and hands
+report.pregnancy_danger_sign.danger_signs.breathlessness = Breathlessness
+report.pregnancy_danger_sign.danger_signs.custom_translations = Custom translations
+report.pregnancy_danger_sign.danger_signs.custom_woman_label_translator = Woman Label translator
+report.pregnancy_danger_sign.danger_signs.custom_woman_label = Woman label
+report.pregnancy_danger_sign.danger_signs.custom_woman_start_label_translator = Woman Start Label translator
+report.pregnancy_danger_sign.danger_signs.custom_woman_start_label = Woman Start label
+
+report.pregnancy_home_visit.context_vars =  Context Variables
+report.pregnancy_home_visit.context_vars.lmp_date_8601_ctx = LMP
+report.pregnancy_home_visit.context_vars.lmp_method_approx_ctx = LMP  Approximate
+report.pregnancy_home_visit.context_vars.pregnancy_follow_up_date_recent_ctx = Recent Pregnancy Follow Up Date
+report.pregnancy_home_visit.context_vars.risk_factors_ctx = Risk Factors
```

Comment:
Should either of both of these be sentence case?

Reply:
The `context_vars` group is hidden from the report view, so will be removed from the translations.